pkg/userdata/generate/controlplane.go

```diff
@@ -16,6 +16,9 @@ security:
     ca:
       crt: "{{ .Certs.K8sCert }}"
       key: "{{ .Certs.K8sKey }}"
+    etcd:
```

Comment:
I know this is all kind of temporary, but etcd should not be under kubernetes.  Both topologically and from a PKI standpoint, it should be a separate top-level category.

Further, etcd has two PKI groups:  peering and client-server, and even if we use the same for both right now, the userdata should reflect these distinctions (even if it is just a subcategorization).

So, for instance, I would use security.etcd.peer.ca and security.etcd.service.ca and default the service PKI to peer, if it is not defined.

Reply:
+1

Reply:
think this is addressed in the latest commit. 
```
security.etcd.ca.key // security.etcd.ca.crt
```